Octokit.Reactive/Clients/IObservableWatchedClient.cs

```diff
@@ -11,24 +17,41 @@ public interface IObservableWatchedClient
         /// <param name="owner">The owner of the repository</param>
         /// <param name="name">The name of the repository</param>
         /// <exception cref="AuthorizationException">Thrown if the client is not authenticated</exception>
-        /// <returns>A <see cref="IObservable{User}"/> of <see cref="User"/>s watching the passed repository</returns>
```

Comment:
Why are we removing the return description? If that's a direction we want to take, might as well remove the entire `/// <returns></returns>` line.